BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -444,7 +444,7 @@ public final class ConvivaAnalytics: NSObject {
     }
 
     private func setVastAdMetadata(adInfo: inout [String: Any], vastAdData: VastAdData) {
-        adInfo["c3.ad.description"] = vastAdData.description
+        adInfo["c3.ad.description"] = vastAdData.adDescription
```

Comment:
Fixing wrong value being used for `c3.ad.description`.

Note: `.description` is a debug print of the object usually on iOS, therefore the actual description `String` is stored in the `adDescription` property.